src/components/AllServicesDropdown/AllServicesTabs.tsx

```diff
@@ -7,6 +7,12 @@ import StarIcon from '@patternfly/react-icons/dist/dynamic/icons/star-icon';
 import { FAVORITE_TAB_ID, TAB_CONTENT_ID } from './common';
 import type { AllServicesSection as AllServicesSectionType } from '../AllServices/allServicesLinks';
 import { isBeta } from '../../utils/common';
+import { Divider } from '@patternfly/react-core/dist/esm/components/Divider';
+import { Text, TextVariants } from '@patternfly/react-core/dist/esm/components/Text';
+import ChromeLink from '../ChromeLink';
+import { Button } from '@patternfly/react-core';
```

Comment:
Use absolute import path for dependency sharing optimization purposes.

Reply:
fixed